modules/client-configuration/pages/tshoot-clients.adoc

```diff
@@ -121,37 +121,9 @@ salt --batch-size 50 '*' state.sls mgr_start_event_grains
 
 
 
-== SSL errors
-
-On SLES{nbsp}11 systems, clients can sometimes have SSL errors which make some operations unusable, including package management and bootstrapping.
-In this case, you will see an error like this:
-
-----
-Repository 'SLES11-SP4-SUSE-Manager-Tools x86_64' is invalid.
-[|] Valid metadata not found at specified URL(s)
-Please check if the URIs defined for this repository are pointing to a valid repository.
-Skipping repository 'SLES11-SP4-SUSE-Manager-Tools x86_64' because of the above error.
-Download (curl) error for 'www.example.com':
-Error code: Unrecognized error
-Error message: error:1409442E:SSL routines:SSL3_READ_BYTES:tlsv1 alert protocol version
-----
-
-This occurs because Apache requires TLS{nbsp}v1.2, but older versions of SLES do not support this version of the TLS protocol.
-To fix this error, you need to force Apache to accept a greater range of protocol versions.
-Open the [path]``/etc/apache2/ssl-global.conf`` configuration file, locate the [systemitem]``SSLProtocol`` line, and update it to read:
-
-----
-SSLProtocol all -SSLv2 -SSLv3
-----
-
-This will need to be done manually on the server, and with a Salt state on the Proxy.
-Restart the [systemitem]``apache`` service on each system after making the changes.
-
-
-
 == Adding Additional FQDNS for Proxies
 
-You have clients connecting through a {productname} proxy. Although these clients may appear in the {webui} you are unable to see that they are connected through a proxy. If this is the case you could be using a FQDN to connect to the proxy that is unknown to {productname}. 
+You have clients connecting through a {productname} proxy. Although these clients may appear in the {webui} you are unable to see that they are connected through a proxy. If this is the case you could be using a FQDN to connect to the proxy that is unknown to {productname}.
```

Comment:
```suggestion
You have clients connecting through a {productname} proxy. Although these clients may appear in the {webui} you are unable to see that they are connected through a proxy.
If this is the case you could be using a FQDN to connect to the proxy that is unknown to {productname}.
```
If you really want to touch this here, you can also fix the other white-space issue :)

Reply:
Good spot, Karl!